src/main/content/_assets/css/toc.scss

```diff
@@ -487,7 +488,8 @@
     }
 
     #toc_inner {
-        width: inherit;
+        height: calc(100vh - 101px);
+        overflow: auto;
```

Comment:
We may want to change it to overflow-y: auto only since on Safari it has a horizontal scrollbar at full resolution:
![image](https://user-images.githubusercontent.com/6392944/64701681-d82f9980-d46e-11e9-9abd-642a3464473c.png)